python/ray/serve/_private/deployment_scheduler.py

```diff
@@ -211,6 +235,18 @@ class LaunchingReplicaInfo:
     target_labels: Optional[Dict[str, Any]] = None
 
 
+def _flatten(
+    deployment_to_replicas: Dict[DeploymentID, Dict[str, Any]]
+) -> Dict[ReplicaID, Any]:
+    """Flattens a dict of {deployment_id: {replica_id: val}} to {replica_id: val}."""
+
+    return {
+        replica_id: val
```

Comment:
just to double confirm that replica_id is globally unique not only unique within a deployment

Reply:
Yup!